src/main/java/io/codelex/flightplanner/flights/Flight.java

```diff
@@ -0,0 +1,101 @@
+package io.codelex.flightplanner.flights;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class Flight {
+
+    private int id;
+    private Airport from;
+    private Airport to;
+    private String carrier;
+    private LocalDateTime departureTime;
+    private LocalDateTime arrivalTime;
+
+
+    public Flight() {
+
+    }
+
+    public boolean doExistSearchFlightRequest(SearchFlightsRequest searchFlightsRequest){
```

Comment:
Šis arī tāds interesants nosakums - 
`public boolean flightMatches(SearchFlightsRequest searchFlightsRequest){`
Un pēc tā jau ir skaidrs, ka pārbaudam vai lidojums sakrīt ar request'u